src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -47,27 +58,21 @@ jit_abs_emitter::jit_abs_emitter(dnnl::impl::cpu::aarch64::jit_generator* host,
 
 size_t jit_abs_emitter::get_inputs_count() const { return 1; }
 
-void jit_abs_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
-    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
-        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
-    } else {
-        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
-    }
-}
-
-template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa, typename type>
```

Comment:
Could you please compare binary size for this fix?

Reply:
size: **50,986,192** (branch) vs **50,967,360** (master) bytes, diff: **18,832** bytes

OS version:
```
% sw_vers
ProductName:		macOS
ProductVersion:	14.2.1
BuildVersion:		23C71
```
generate & build:
```
% cmake -DCMAKE_BUILD_TYPE=Release ..
% cmake --build . --parallel 12
```

**branch es/cpu/eltwise_arm_fp16 (beb955000741d61ee9d51f9ddf84011019a971ab)**
```
% ls -lt ../bin/arm64/Release/libopenvino_arm_cpu_plugin.so
-rwxr-xr-x  1 eshoguli  staff  50986192 20 May 14:51 ../bin/arm64/Release/libopenvino_arm_cpu_plugin.so
```
**reference master (7061e92a2d97773ca4ca145d4ad3ad46df727298)**
```
-rwxr-xr-x  1 eshoguli  staff  50967360 20 May 15:09 ../bin/arm64/Release/libopenvino_arm_cpu_plugin.so
```